tests/manage/storageclass/test_rbd_csi_default_sc.py

```diff
@@ -64,11 +28,35 @@ class TestBasicPVCOperations(ManageTest):
     """
 
     def test_ocs_347(self):
-        global PVC, STORAGE_CLASS
         log.info("Creating RBD StorageClass")
-        STORAGE_CLASS = helpers.create_storage_class(
-            constants.CEPHBLOCKPOOL, POOL.name, SECRET.name
+        sc_obj = helpers.create_storage_class(
+            interface_type=constants.CEPHBLOCKPOOL,
+            interface_name=self.cbp_obj.name,
+            secret_name=self.rbd_secret_obj.name,
         )
 
         log.info("Creating a PVC")
-        PVC = helpers.create_pvc(sc_name=STORAGE_CLASS.name)
+        pvc_obj = helpers.create_pvc(
+            sc_name=sc_obj.name, wait=True,
+        )
+
+        log.info(
+            f"Creating a pod on with pvc {pvc_obj.name}"
+        )
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL, pvc_name=pvc_obj.name,
+            desired_status=constants.STATUS_RUNNING, wait=True,
+            pod_dict_path=constants.NGINX_POD_YAML
+        )
+
+        log.info("Deleting Pod")
```

Comment:
try moving the deletion of resources to the teardown

Reply:
pvc_obj and other objects are not available outside the class since I created these as part of my test. Previously I was using global variable to workaround this.

Reply:
@parikshithb you can do it as done in the resources fixtures in https://github.com/red-hat-storage/ocs-ci/pull/516/files

Reply:
ack